css/properties.json

```diff
@@ -4712,6 +4640,49 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "offset": {
+    "syntax": "&lt;'offset-path'&gt; &amp;&amp; &lt;'offset-distance'&gt; &amp;&amp; &lt;'offset-rotation'&gt; &amp;&amp; &lt;'offset-anchor'&gt;",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
```

Comment:
That's incorrect. All five longhands are animatable, so this value should be an array listing all the longhands.

Sebastian